src/hotspot/cpu/riscv/c1_LIRAssembler_arraycopy_riscv.cpp

```diff
@@ -57,7 +57,7 @@ void LIR_Assembler::generic_arraycopy(Register src, Register src_pos, Register l
   __ mv(c_rarg4, j_rarg4);
 #ifndef PRODUCT
   if (PrintC1Statistics) {
-    __ add_memory_int32(ExternalAddress((address)&Runtime1::_generic_arraycopystub_cnt), 1);
+    __ incrementw(ExternalAddress((address)&Runtime1::_generic_arraycopystub_cnt));
```

Comment:
Is it really `incrementw`, though? These counter fields are `int`-s, are they still 32-bit on RISC-V? If so, shouldn't it be `incrementl`?

Reply:
@shipilev : Yes, the type of _generic_arraycopystub_cnt is int and it occupies 32-bit in memory. That's why we use incrementw here which increments a 32-bit memory operand. Note that incrementl works for 64-bit memory operand. Hope that explains. Thanks.

Reply:
Is this RISC-V specific postfix naming? On x86, there is `b`/`w`/`l`/`q` for 1/2/4/8-byte ops, respectively. This is just my curiosity, it does not block the integration.

Reply:
Yes, default is for 8-byte ops and we use 'w' post-fix for 4-byte ops on RISC-V. And I think this is hangover from the aarch64 port [1]. 

[1] https://github.com/openjdk/jdk/blob/master/src/hotspot/cpu/aarch64/macroAssembler_aarch64.cpp#L1917